engine/src/main/java/org/hibernate/search/engine/impl/ConfigContext.java

```diff
@@ -157,12 +158,9 @@ private FilterDef interpretFullTextFilterDef(FullTextFilterDef defAnn) {
 			return filterDef;
 		}
 		try {
-			filterDef.getImpl().newInstance();
+			ClassLoaderHelper.untypedInstanceFromClass( filterDef.getImpl(), "filter definition" );
 		}
-		catch (IllegalAccessException e) {
-			throw new SearchException( "Unable to create Filter class: " + filterDef.getImpl().getName(), e );
-		}
-		catch (InstantiationException e) {
+		catch (Exception e) {
```

Comment:
This really should catch `RuntimeException`: there's no reason to catch `Exception` if no checked exception is thrown, and this way we will have a compile-time error if we ever add a checked exception to `ClassLoaderHelper.untypedInstanceFromClass`.

Reply:
ok.